src/main/java/teammates/ui/controller/Action.java

```diff
@@ -576,10 +584,131 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
                && loggedInUser.googleId != null
                && !loggedInUser.googleId.equals(requestedUserId);
     }
+
+    /**
+     *Common method to Get the feedback data
+     */
+
+    protected FeedbackSessionAttributes extractFeedbackSessionData() {
+
+        FeedbackSessionAttributes newSession = new FeedbackSessionAttributes();
+
+        newSession.setCourseId(getRequestParamValue(Const.ParamsNames.COURSE_ID));
+        newSession.setFeedbackSessionName(Sanitizer.sanitizeTitle(
+                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME)));
+
+        // getting the class name from which it's being called
+        String className = this.getClass().getName();
+
+        if ("InstructorFeedbackEditSaveAction".equals(className)) {
```

Comment:
@thenaesh 
>can be extracted into a smaller method, which does newSession.setCreatedTime(new Date()) in the base class

  These atrributes has to created one after other  means - before doing `newSession.setCreatorEmail()`  say i have to do `setCreateTime()` ?. If that doesn't affect the functionality of the method then i can put those attributes in the subclass itself.

>This method can then be called from extractFeedbackSessionData, and binds to either the base class version or the subclass version **depending on the type of the object from which it is being called.**

Again to check the type of the object called i have to use the classname right ?